FireRender.Maya.Src/FireRenderVolume.cpp

```diff
@@ -430,7 +430,7 @@ bool NorthstarRPRVolume::TranslateVolume()
 	m_emissionGrid.Reset();
 
 
-	if (vdata.densityGrid.IsValid()) // grid exists
+	if (vdata.densityGrid.IsValid() && vdata.densityGrid.gridOnIndices.size() > 0) // grid exists
```

Comment:
Please consider moving check gridOnIndices.size() > 0 to IsValid() method.

If I understood correctly grid with gridOnIndices.size() == 0 would be invalid, right?